### ISSUE TITLE
Footer Component

### ISSUE THREAD

**Issue by matthewweale** [Underdog-Devs/website#95]
## Summary

Minor changes need to be made to the Footer in order for launch

## Acceptance Criteria

- Fix alignment / spacing to match screenshot
- Remove "About", "Terms", "Privacy"
- Change "p" tags to "buttons"
- Add on clicks to all of the buttons that open up to a random website for now. We will replace the link with a coming soon page once the coming soon page is complete
- Styling updates should be made in src/CSS/Footer.module.css
- Component should be built in /src/Components/Footer.js
- Code must compile and generate no new warkings
- Attach a screenshot of the completed work to this ticket as part of the submission

## Discussion

This is a collaborative project so please contact a project lead before updating or changing any project dependencies. NPM is the package manager of choice for this project.

Tickets are a living document. Please ask any questions, make any comments, any notes for future work on this ticket for discussion and record keeping purposes.

## Screenshots

![Screen Shot 2021-03-22 at 1 43 07 PM](https://user-images.githubusercontent.com/50917450/112055805-8a402b00-8b14-11eb-8f3a-7fc46d85f0e4.png)

## Additional Information

Expected completion date here:
